libtransmission/torrent-files.cc

```diff
@@ -326,6 +326,11 @@ namespace
 // for example, NUL.txt is not recommended.
 [[nodiscard]] bool isReservedFile(std::string_view in) noexcept
```

Comment:
`[[maybe_unused]] std::string_view in` and then you can drop the `(void)in;` below